docs/components/sensor/bme280.md

```diff
@@ -5,7 +5,8 @@ weight: 70
 type: "docs"
 description: "Configure a bme280 sensor."
 tags: ["sensor", "components"]
-icon: "img/components/sensor.png"
+icon: "img/components/sensor.svg"
+images: ["/components/img/components/sensor.svg"]
```

Comment:
Nice catch! But will `icon:` resolve from `sensor` directory? As far as I understand, this relative path expects `docs/components/sensor/img/components/sensor.svg` to exist, but I think we mean `docs/components/img/components/sensor.svg`. The full path provided to `images:` looks like it should work for `icon:` as well! This also applies to the other places you updated the `icon:` field.

I can see this confusion because `docs/components/sensor/img/` _does_ exist, but contains the specific sensor images, not the generic landing page one.